CHANGELOG.md

```diff
@@ -3,10 +3,11 @@
 ## 2.3.0 (Unreleased)
 
 ## Changed
+* [ALFREDAPI-368](https://xenitsupport.jira.com/browse/ALFREDAPI-368): Replace obsoleted Gradle plugins (org.dm.bundle and ampde)
```

Comment:
It's important that the order stays chronologically. Can you put it back?

Reply:
Isn't it chronologically now? I thought the ticket numbers were ordered form low to high.

Reply:
> Isn't it chronologically now? I thought the ticket numbers were ordered form low to high.

@kpiot123 I assume it is chronologically ordered based on implementation date and not on ticket creation date.

Reply:
I don't think that is very practical if you try to read the changelog.

Reply:
Chronological ordered on commit date. That's how all our changelog are. They mirror the git log.

Reply:
> Chronological ordered on commit date. That's how all our changelog are. They mirror the git log.
> Just put it back the way it was, guys :) There was never any reason to mess with the existing order anyway

Ok. The order was reverted in my last commit.